### DISCUSSION TITLE
music/rode/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/music#5] [General]
# music/rode/

http://shanenull.com/music/rode/

**Comment by shane0** [comment]
https://rode.com/en-us/user-guides/ai-micro

**Reply by shane0** [reply]
https://rode.com/en-us/apps/rode-central

**Reply by shane0** [reply]
when this breaks replace it with something with a dac and something without this tech lock in nonsense this thing only works with special rode cables they are not standard usb what the hell kind of toxic corporate crap is this? lol trolls

**Comment by shane0** [comment]
https://rode.com/en-us/interfaces-and-mixers/ai-series/ai-micro?srsltid=AfmBOook7vrAX6yX0fTZbL9u6gVX5q4gT9PwzytA5Bdcm28ZkPMHoyie
